src/js/DevContest.js

```diff
@@ -1,6 +1,7 @@
 /* Web3 Setup */
 var Web3 = require('web3');
 
+/* Create web3 for Metamask interaction */
 if (typeof web3 !== 'undefined') {
   web3 = new Web3(web3.currentProvider);
```

Comment:
Global variables are a no no, create a scope and then pull the variables from that scope.